### DISCUSSION TITLE
How to test/use etcd client locally?

### DISCUSSION THREAD

**Discussion by shreemaan-abhishek** [etcd-io/etcd#14737] [Q&A] [answered]
Hi, how do I use the etcd client locally? I was tinkering with the client trying to understand how it works and what are things going on under the hood. So I made a few code changes and used the following command:

`go mod edit -replace go.etcd.io/etcd/client/v3=../etcd/client/v3`

But this did not work appropriately. For example, when I had to use `"go.etcd.io/etcd/client/v3/concurrency"` the above command did no good as it was still downloading the packages from the internet instead of depending on the local one.

**Comment by SimFG** [comment, marked as answer]
Is there any error message? Maybe you need more some `replace`, because the `client/v3` depends on `go.etcd.io/etcd/api/v3` and `go.etcd.io/etcd/client/pkg/v3`.
![image](https://user-images.githubusercontent.com/21985684/202127758-ebfb7f7b-3fa0-4b35-8609-18bd5aed923d.png)

**Reply by shreemaan-abhishek** [reply]
Thank you so much. This helped!!
